libs/ledger/src/chain/block_coordinator.cpp

```diff
@@ -319,11 +319,11 @@ BlockCoordinator::State BlockCoordinator::OnSynchronising()
   FETCH_UNUSED(current_dag_epoch);
 
   // initial condition, the last processed block is empty
-  if (chain::GENESIS_DIGEST == last_processed_block)
+  if (chain::ZERO_HASH == last_processed_block)
```

Comment:
I suspect that these should probably be genesis digest. I think that in the non-proof of stake configuration we should probably update the genesis block hash too